test/src/main/java/org/apache/accumulo/test/ScanConsistencyIT.java

```diff
@@ -79,12 +79,57 @@
 public class ScanConsistencyIT extends AccumuloClusterHarness {
 
   private static final Logger log = LoggerFactory.getLogger(ScanConsistencyIT.class);
+  private static boolean inTestingContext;
+
+  public static void main(String[] args) {
+    /**
+     * @formatter:off
+     * Note: In order to run main,
+     * 1) Build the project
+     * 2) Copy the accumulo test jar (in /test/target/) to your accumulo installation's
```

Comment:
I think we could further minimize changes to the test by requiring copying any needed dependencies like junit

```
mvn dependency:copy-dependencies -DincludeGroupIds="org.junit.jupiter"
cp test/target/dependency/junit-jupiter-* $ACCUMULO_HOME/lib/
```

If that works, then would not need the wrappers around the junit assert method call, which would minimizes the test changes.